### DISCUSSION TITLE
change browse word in cds-file

### DISCUSSION THREAD

**Discussion by robertIsaac** [vmware-clarity/core#50] [Q&A] [answered]
we are using our custom localization, and I want to know how to change the browse word in the cds-file

**Comment by ashleyryan** [comment, marked as answer]
Here's an example using the I18nService: https://stackblitz.com/edit/clarity-angular-demo-sc4ck4?file=src%2Fapp%2Fapp.component.ts

It's [documented here](https://clarity.design/storybook/core/?path=/story/foundation-internationalization--page) and you can see all of the [default translations and the object structure here](https://github.com/vmware-clarity/core/blob/b291924a832dcda87e0a7a8cb349b3245aa986a4/projects/core/src/internal/services/i18n.service.ts#L23)
